src/internal_rules.jl

```diff
@@ -750,12 +742,11 @@ function EnzymeRules.augmented_primal(
         func::Const{typeof(ldiv!)},
         RT::Type{<:Union{Const, DuplicatedNoNeed, Duplicated, BatchDuplicatedNoNeed, BatchDuplicated}},
 
-        A::Annotation{AType},
-        B::Union{Const, DuplicatedNoNeed, Duplicated, BatchDuplicatedNoNeed, BatchDuplicated};
+        A::Annotation{<:Cholesky},
+        B::Union{Const{<:AbstractVecOrMat}, DuplicatedNoNeed{<:AbstractVecOrMat}, Duplicated{<:AbstractVecOrMat}, BatchDuplicatedNoNeed{<:AbstractVecOrMat}, BatchDuplicated{<:AbstractVecOrMat}};
```

Comment:
Same here, this should just be B

Reply:
It's
```suggestion
        B::Union{Const, DuplicatedNoNeed, Duplicated, BatchDuplicatedNoNeed, BatchDuplicated};
```
on main, is this incorrect?

Reply:
ah I see what main is doing here, it's doing Any annotation, except active. You should leave the main variant.

I wouldn't specialize the abstractvecormat here.

Reply:
👍 Should the same (without AbstractVecOrMat) be used also for the `forward` rule?

Reply:
No, because Active is only applicable to reverse mode, so all activities are valid there.